src/main/groovy/io/protop/gradle/plugin/ProtopGradlePlugin.groovy

```diff
@@ -0,0 +1,87 @@
+package io.protop.gradle.plugin
```

Comment:
I like this so much better than the Java version

Reply:
Should we move everything over to Groovy for consistency? An alternative would be Kotlin but I have zero experience with that language (I'm willing to learn but it'll slow down progress).